ControlFlow/Program.cs

```diff
@@ -48,7 +63,23 @@ public static string LandscapeOrPortrait(int width, int height)
         // Suspended" instead. Don't Worry about input validation.
         public static string SpeedTrap(int speedLimit, int speedCar)
         {
-            return default;
+            var SD = speedCar - speedLimit;
+            var DS = 0;
```

Comment:
I would always recommend naming your variables in plain English or in Pascal case (pascalCase).
This way when other programmers are reviewing your code they can make sense of the variables and their purpose a lot faster.